docs/source/recipes/appextras.md

```diff
@@ -23,14 +27,16 @@ Or you can use the new key of `config.settings`, the `appExtras`. This is
 a list of registrations, each registration is an object with:
 
 - `match`: The `match` key is for objects compatible with [react-router's
```

Comment:
Let's combine lines, and comply with Plone's one sentence per line for narrative documentation style.

```suggestion
- `match`: The `match` key is for objects compatible with [react-router's `matchPath`](https://v5.reactrouter.com/web/api/matchPath). It can be either a string or a [`match`](https://v5.reactrouter.com/web/api/match) object.
```